### PR TITLE
Add submit button validation

### PR DESCRIPTION
**Pull request by Salimer**
In this milestone, I've tried to use client-side validation to catch and throw errors in the UI by checking the following conditions:

- The email field has to be in lowercase.
- When the user submits the form, I check if the email is in lowercase using javascript.
- If the validation is OK, the form is sent.
- If the validation is not OK, show an error message to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -11,3 +11,23 @@ navItems.forEach((element) => {
...
+  if (!emailInput.validity.valid) {
+    errorElement.textContent = '*Please enter the required information.';
+    event.preventDefault();
+  }
+  setTimeout(() => {
+    errorElement.textContent = '';
+  }, 3000);
+}
+
+submit.addEventListener('click', showError);
```
Comment (Qoosim) [inline script.js:33]: - [ ] KIndly ensure that the user is able to submit the form after every validation conditions is met. At this moment, I fulfilled all the conditions, but the form refused to submit after clicking on the `Get in touch` button. Please, attend to that.
[app-behave.webm](https://user-images.githubusercontent.com/26597432/231480317-215d26bd-aac5-4f23-8d2c-b15fa71e35b1.webm)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Salimer) [reply script.js:33]: > [ ] KIndly ensure that the user is able to submit the form after every validation conditions is met. At this moment, I fulfilled all the conditions, but the form refused to submit after clicking on the `Get in touch` button. Please, attend to that.
> [app-behave.webm](https://user-images.githubusercontent.com/26597432/231480317-215d26bd-aac5-4f23-8d2c-b15fa71e35b1.webm)

Thank you very much @Qoosim for the feedback. Unfortunately the link you shared didn't work for me!

**Review by OyePriscilla — APPROVED** [review]
Hi @Salimer and @kifle23 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
 


## Highlights
Contact form contains validation checking if the email is written in all lowercase ✔️ 
Error message display when email is not written in lowercase ✔️ 
Use JS event listener ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**